Dockerfile.testdb

```diff
@@ -0,0 +1,27 @@
+FROM mysql:5.7
```

Comment:
Is there any way for this to be combined with the existing Dockerfile.MySQL too?